### DISCUSSION TITLE
Error 500 (JSON exception when decoding response: Syntax error) using Firefly III Importer v1.2.2 and YunoHost v11.1.18

### DISCUSSION THREAD

**Discussion by jdenozi** [firefly-iii/firefly-iii#7398] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I have generated the OAuth credential, I then go to FireFly III Data import v1.2.2. I put my url and the client id.  I arrive on a page which asks me an authorization for firefly III v6.0.5 and that I would be redirected to xxxx. By clicking on authorize I fall on an error 500 (JSON exception when decoding response: Syntax error )

### Debug information

Debug information generated at 2023-04-18 16:57:32 Europe/Amsterdam for Firefly III Data Importer version **1.2.2**.

| Scope | Version |
| --- | --- |
| FIDI | 1.2.2 |
| Build | (unknown), (unknown) |
| PHP | 8.2.5 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Using docker? | false |
| System TZ | Europe/Amsterdam |
| Browser TZ | Europe/Paris |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | fpm-fcgi |
| BCscale | 12 |
| Trusted proxies (.env) | * |

| User info | Value |
| --- | --- |
| User agent | Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0 |

### Expected behaviour

I just want to be able to use the data importer to use FireFly

### Steps to reproduce

1. Download YuhoHost v11.1.18
2. Setup with yunohost domain (xxxx.ynh.test) 
3. Install FireFly with one admin with YunoHost
4. Create the first account with personal mail address 
5. Create the OAuth 
6. Download FireFly III Data Importer v1.2.2
7. Enter url firefly information and client ID and submit
8. Authorize 

### Additional info

I had other problems that I could solve before like parsing problems and I had to change the local language. Then an error with the version of php which was version 7 instead of the necessary 8

**Comment by JC5** [comment]
What's the error exactly? 500 is a generic message. Anything in your logs?

**Comment by jdenozi** [comment]
> What's the error exactly? 500 is a generic message. Anything in your logs?

The error is: Json Exception when decoding response: Syntax error). Well nothing in the log of the Firefly III and only this in Importer 


[log.txt](https://github.com/firefly-iii/firefly-iii/files/11264624/log.txt)

**Comment by JC5** [comment]
I see that Yunohost is blocking access to Firefly III, instead of seeing JSON, HTML from Yunohost is returned. There's not much I can do about that.
